### ISSUE TITLE
Suggestion - make session locking more flexible

### ISSUE THREAD

**Issue by greecemunky** [MonstaApps/Monsta-FTP#69]
I think the ability to make the IP address range adjustable so either the end user or Monsta FTP site administrator can specify how "loose" session locking is.

One use case was a client who I had locked down.  Their system appeared to use a load balanced over 2 or so adjacent IP addresses - HTTP requests would bounce between the two.  As such, I had to disable session locking.

The CIDR mask would be a good candidate for this loose/variable session locking.  In the above scenario, if I could, I would have opted for a /24 mask rather than having it open to all.

To implement, for example, a variable named $sessLockMask can hold the value e.g. 24 (range 0-32; default value of 32 for current session locking, 0 to disable session locking) which can then be used for checks later within the code.

**Comment by MonstaApps** [comment]
Yes, I agree. I will add this to the next release (before v2.0 drops).

**Comment by MonstaApps** [comment]
Would it be good enough for the rule to be "IP must start with xxx"
So in config.php you would have:

$ipLock = "192.168.0";

Then the PHP code would to a preg_match on the user's IP.
